pkg/lib/flavor/util/platform_flavor_util.go

```diff
@@ -210,6 +211,8 @@ func (pfutil PlatformFlavorUtil) GetHardwareSectionDetails(hostInfo *taModel.Hos
 		// Set TPM Feature presence
 		tpm := fm.TPM{}
 		tpm.Enabled = hostInfo.HardwareFeatures.TPM.Enabled
+		tpm.Supported = hostInfo.HardwareFeatures.TPM.Supported
```

Comment:
@isteffyx Can we eliminate the duplicate "hostinfo" structures in flavor vs. model?